common.go

```diff
@@ -271,6 +292,12 @@ func setupAuthTLS(auth authConfig, saramaCfg *sarama.Config) error {
 	return nil
 }
 
+func qualifyPath(argFN string, target *string) {
+	if *target != "" && !filepath.IsAbs(*target) {
```

Comment:
i think we need to also check if `filepath.Dir(*target) == "."` -- cf test failures for `test-secrets/cert`.